lib/config/configuration_test.go

```diff
@@ -778,6 +778,9 @@ func TestApplyConfig(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, types.AgentMesh, tunnelStrategyType)
 	require.Equal(t, types.DefaultAgentMeshTunnelStrategy(), cfg.Auth.NetworkingConfig.GetAgentMeshTunnelStrategy())
+	require.Equal(t, cfg.Auth.Preference.Origin(), types.OriginConfigFile)
```

Comment:
These assertions are backwards. The expected value should come before the actual value.

Reply:
Thanks! Done in 3a7053dd7864b6193f43cb4201d0af7eaef03381.